tensorflow/python/compiler/tensorrt/utils.py

```diff
@@ -69,3 +73,42 @@ def is_linked_tensorrt_version_greater_equal(major, minor=0, patch=0):
 def is_loaded_tensorrt_version_greater_equal(major, minor=0, patch=0):
   ver = _pywrap_py_utils.get_loaded_tensorrt_version()
   return _is_tensorrt_version_greater_equal(ver, (major, minor, patch))
+
+
+def is_experimental_feature_activated(feature_name):
+  """This helper function checks if an experimental feature was enabled using
+  the environment variable `TF_TRT_EXPERIMENTAL_FEATURES=feature_1,feature_2`.
+
+  Args:
+    feature_name: Name of the feature being tested for activation.
+  """
```

Comment:
A doc string should start with a one-line summary, see [here](https://google.github.io/styleguide/pyguide.html#381-docstrings)

Reply:
Fixed